docs/docs/recipes/working-with-images.md

```diff
@@ -385,12 +391,12 @@ export const pageQuery = graphql`
 // highlight-end
 ```
 
-4. Run `gatsby develop`, which will generate images for files sourced in the filesystem
+4. Exécutez `gatsby develop`, qui générera des images pour les fichiers provenant du système de fichiers
 
-### Additional resources
+### Ressources additionnelles
 
-- [Example repository using this recipe](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipes-gatsby-image)
-- [Featured images with frontmatter](/docs/working-with-images-in-markdown/#featured-images-with-frontmatter-metadata)
-- [Gatsby Image API](/docs/gatsby-image/)
-- [Using Gatsby Image](/docs/using-gatsby-image)
-- [More on working with images in Gatsby](/docs/working-with-images/)
+- [Exemple de référentiel utilisant cette recette](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipes-gatsby-image)
+- [Images en vedette avec frontmatter](/docs/working-with-images-in-markdown/#features-images-with-frontmatter-metadata)
```

Comment:
Il faut aussi traduire les ancres, peux-tu confirmer @mindsers ?

Reply:
Hello @SamuelAlev for the recent code changes that you gave, you want me to keep the code as it was before, which was in french. Also in my last commit you mentioned to me that JS still don't understand French yet ;) I dont understand how to go forward with this, could you please make it a bit more clear, also @mindsers could you please confirm that we need anchors translated to french or not. Furtherly i would request @dantereve @gatarelib @alexislepresle to review my work ASAP